.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: node_js
 node_js:
-  - 10
-  - 8
-  - 6
+  - "10"
```

Comment:
please leave CI up to maintainers and revert these changes :)

Reply:
understood, but that's still a CI commit on a docs PR for an error that's only being displayed on your end :)